### ISSUE TITLE
Feature req: "quote words" helper

### ISSUE THREAD

**Issue by papandreou** [gustavnikolaj/string-utils#1]
Hey, how about a [`qw` helper](https://perlmaven.com/qw-quote-word) for old, disgruntled perl programmers? :laughing:

```js
const qw = (strs, ...args) =>
  zip(strs, ...args).join('').split(/\s+/).filter(item => item);

const blah = 'hey';
console.log(qw`   foo   bar
quux   ${blah}   baz
`); // [ 'foo', 'bar', 'quux', 'hey', 'baz' ]
```

**Comment by gustavnikolaj** [comment]
Sure :-)

I started this repo to avoid having to repeat myself and copy/paste utilities around. I'm working on internal tooling these days (translation tools, build tools etc) and I'm trying to focus a lot on the quality of output, so the current utilities are focused around that. 

For one of the less sane things, see reflow that I just added in v1.1.0: https://github.com/gustavnikolaj/string-utils#reflow

Out of interest, what is your use case for qw? I'm not familiar with it, but I get a taste of shell in my mouth when looking at it. I have a utility for that too, that I'm not quite sure I want to put out there (I'm afraid of what people will think of me...)

```js
      shFormat`
        ssh -o 'StrictHostKeyChecking no' ${this.host} "
          sudo systemctl show chef-client -p ActiveState
          && sudo systemctl show chef-client -p SubState
        "
      `
```

Automatically escaping new lines to make authoring long shell commands more comfortable in node. I think you can roughly guess what I'm working on from the above example.

I use it in combination with another function:

```js
import { exec } from "child_process";

export const shFormat = (strs, ...args) =>
  strs
    .map((str, index) => {
      if (args[index]) {
        str = str + args[index];
      }
      return str.replace(/\n/g, "");
    })
    .join("");

export default function runShellCommand(command, { cwd }) {
  return new Promise((resolve, reject) => {
    exec(command, { cwd }, (err, stdout, stderr) =>
      resolve({ err, stderr, stdout })
    );
  });
}
```

Those two tools together make it really really comfortable integrating with existing tools over ssh :-)

**Comment by papandreou** [comment]
The use case for the `qw/.../` operator in perl is to make it easier to declare array literals of single word strings. It’s kinda handy, but would probably raise too many eyebrows in a JS context. I’m not really sure 😆

**Comment by gustavnikolaj** [comment]
Released in v1.2.0 :-)
